libs/feature/search/src/lib/results-table/results-table.component.html

```diff
@@ -0,0 +1,135 @@
+<div>
+  <div class="flex records-information">
+    <div
+      translate
+      class="my-8 ml-4 block text-gray-800"
+      [translateParams]="{
+        displayed: (records$ | async).length,
+        hits: (recordCount$ | async)
+      }"
+    >
+      results.records.hits.displayedOn
+    </div>
+    <div
+      *ngIf="(selectedRecords$ | async).length as selectedRecordCount"
+      class="my-8 ml-6 block text-gray-400 selected-records"
+      translate
+      [translateParams]="{ amount: selectedRecordCount }"
```

Comment:
To discuss: how does translation work on this project? I will need it for the rich text editor component.

Reply:
Discussed separately; for the record, here is the doc about i18n in the project: https://geonetwork.github.io/geonetwork-ui/main/docs/reference/i18n.html